arangod/Replication2/ReplicatedLog/Supervision.cpp

```diff
@@ -969,7 +972,9 @@ auto checkConfigCommitted(SupervisionContext& ctx, Log const& log) -> void {
 
   auto const& plan = *log.plan;
   auto const& current = *log.current;
-
+  if (!current.supervision.has_value()) {
+    return;
+  }
   ADB_PROD_ASSERT(current.supervision.has_value());
```

Comment:
NIT, these assertions are now redundant, because of the check right above.

Reply:
Agreed.
I doubt that this will have measurable performance impact, also there are other places exactly like this.
S would like to keep it for now, may improve in future.